person.rb

```diff
@@ -23,6 +24,10 @@ def can_use_services?
     is_of_age || parent_permission
   end
 
+  def add_rental(date, person)
+    @rentals.push(Rental.new(date, self, person))
+  end
```

Comment:
- [ ] The rental method should have `book` and `date` parameters, take  look at the example given below

![image](https://user-images.githubusercontent.com/81932478/216384331-ce6d1cd2-b6c4-4399-b90f-d0e189b174b9.png)